packages/app/src/sandbox/eval/presets/preact-cli/old.js

```diff
@@ -0,0 +1,118 @@
+import babelTranspiler from '../../transpilers/babel';
+import jsonTranspiler from '../../transpilers/json';
+import stylesTranspiler from '../../transpilers/style';
+import sassTranspiler from '../../transpilers/sass';
+import rawTranspiler from '../../transpilers/raw';
+import stylusTranspiler from '../../transpilers/stylus';
+import lessTranspiler from '../../transpilers/less';
+
+import asyncTranspiler from './transpilers/async';
+
+import Preset from '..';
+
+export default function PreactPresetOld() {
```

Comment:
We should add a bit more descriptive name, because `old` can mean anything. Maybe something like `preactV9` or something?

Reply:
Changed to v8 since there was no version 9